GVRf/Framework/src/org/gearvrf/GVRLight.java

```diff
@@ -29,8 +29,49 @@ public GVRLight(GVRContext gvrContext) {
         setDiffuseIntensity(1.0f, 1.0f, 1.0f, 1.0f);
         setSpecularIntensity(1.0f, 1.0f, 1.0f, 1.0f);
         isEnabled = true;
+        name="";       
```

Comment:
null might be a better choice; people likely expect to get null instead of an empty string in case the name has not been set.